### ISSUE TITLE
New Action for Tasks in Schedules: Remove Files

### ISSUE THREAD

**Issue by Maelep** [pterodactyl/panel#3702]
### Is there an existing feature request for this?

- [X] I have searched the existing issues before opening this feature request.

### Describe the feature you would like to see.

Add the `Deletes Files` option in the Action drop down menu when creating a new task that will allow users to delete a list of files.

### Describe the solution you'd like.

This action displays a text box allowing you to list paths to files on the server that you wish to delete. For example:
```
./server/rust/*.map
./server/rust/*.sav
./server/rust/*.sav.1
./server/rust/*.sav.2
./server/rust/player.blueprints.*.db
./server/rust/player.blueprints.*.db-journal
```

### Additional context to this request.

Useful for all games and softwares but here I will focus on the game Rust. Indeed, this game works with a Wipe system which requires the deletion of some files at regular intervals. Unfortunately, it is not possible to do it automatically and therefore requires the presence of an administrator at the time of the Wipe. Thanks to this feature we could solve a part of the problem.

**Comment by Zebotxd** [comment]
I i think alot of sandbox games could have great use of this

**Comment by cozyGalvinism** [comment]
Not just sandbox games, pretty much any nest that produces log files, runs backups or uses a file cache of some sort. It'd be great for clean up. Maybe with either regex or globbing support for the file names? (or both)

**Comment by KOYK** [comment]
Any news on this gem?

**Comment by nuallan89** [comment]
indeed .. anything new here ?

**Comment by parkervcp** [comment]
My rust auto-wipe technically already handles this. See https://github.com/parkervcp/eggs/tree/master/game_eggs/steamcmd_servers/rust/rust_autowipe

This could also be managed in the image when it was handled on startup.

it would just be a variable to specify the files that need to be wiped.

**Comment by DaneEveritt** [comment]
Yes, some eggs handle this, but that isn't a reasonable approach long term (or for users wanting highly custom things that aren't also admins). And no, per usual across the entire Pterodactyl organization I only close issues once fixed.

An issue left open just means it hasn't become important enough or interesting enough for me to look into.

**Comment by KissmanCZ** [comment]
This should be one of the basic features :) any progress on this?

**Comment by Tudorhc5** [comment]
Still no updates on this ?

**Comment by geekgarage** [comment]
I can only +1 this, 110% would love to have access to this through scheduled tasks, as mentioned, cleaning log files, data files for Rust wipes and many other games that don't have a wipe feature but you want to be able to wipe at a regular interval.

**Comment by geekgarage** [comment]
> I can only +1 this, 110% would love to have access to this through scheduled tasks, as mentioned, cleaning log files, data files for Rust wipes and many other games that don't have a wipe feature but you want to be able to wipe at a regular interval.

Why is my comment marked as spam?

**Comment by Zekhap** [comment]
Yeah i would love to do custom Tasks

**Comment by ByteHunter0x1** [comment]
I would also +1 adding this functionality to the already great scheduler!
Would save me having to run manual scripts outside the panel.

**Comment by MihailHavenT** [comment]
Any updates about ?

**Comment by DaneEveritt** [comment]
In the interest of maintainer sanity and cleaning up an enormous backlog of stale feature requests we have moved to requiring all feature requests be discussed within [Discussions](https://github.com/orgs/pterodactyl/discussions/categories/feature-requests). Please consider re-opening this request there if you believe it is still relevant and worth consideration.

See https://github.com/pterodactyl/panel/issues/5491 for additional context.

Thanks.

<sub>_This comment is in the interest of issue tracker maintenance: it does not imply that maintainers are endorsing or dismissing your request._</sub>
